docs/commonly_used_data_types/data_viz_query.md

```diff
@@ -0,0 +1,179 @@
+      "query_tz": "America/Los_Angeles",
+      "output_tz": "America/Los_Angeles",
+      "bin_strategy": "user_defined2",
+      "bin_count": 50
+    }
+```
+### name
+This is the name of idependent varaible you are using.
+
+### time_resolution
+This is optional and is how detailed of a time breakdown you want in the variable the options for time_resolution are as follows:
+* year
+* month
+* week
+* day
+* hour
+* minute
+* second
+
+### query_tz
+This is optional and tells the system what time zone the query is in.
+
+### output_tz
+This is optional and tells the system what time zone to return the data in.
+
+### bin_strategy
+This is optional and tells the sytem how you wish to bin the data.  You have the following options:
+* user_defined2
+* none
+* categorical
+
+### bin_count
+This is optoinal and tels the system how many bins you wish to put the data into.
+
+## time_selection
+This is the time frame you want to grab data from there are two different ways to make this time selection, Relative and Absolute
+
+### Relative Time Selection
+Relative Time Selections goes back from now a certain amount of time based on what you tell it.  The format for this time selection is the following:
+```
+{
+    "time_type": "relative",
+    "relative_start": 7,
+    "relative_unit": "year",
+    "ctime_tz": "America/Los_Angeles"
+}
+```
+#### Time Type
+For a relative time selection this needs to be set to "relative".
+
+#### Relative Start
+The amount of units from now you wish to go back to start your time selection.
+
+#### Relative Unit
+The unit of time you wish to go back from now.  Your options for this are as follows:
+* year
+* month
+* week
+* day
+* hour
+* minute
+* second
+
+#### ctime_tz
+The time zone for your time selection
+
+### Absolute Time Selection
+Absolute Time Selections have a start and end time and will gather data from between the two.  The format for this time selection is as follows:
+```
+{
+  "time_type": "absolute",
+  "start_time": "2023-02-23T08:00:00.000Z",
+  "end_time": "2023-03-01T21:35:35.499Z",
+  "time_zone": "America/Los_Angeles"
+}
+```
+#### Time Type
+For absolute time selecitons this must be set to absolute.
+
+#### Start Time
+The time you wish to start the time selection at.
+
+#### End Time
+The time you wish to end the time selection at.
+
+#### Time Zone
+The time zone for the time selection.
+
+## Where
+This is optional.  It will narrow down the data return based on criteria given. The list will be anded together. This is a list in the following format:
+```
+{
+    "name": "type__part_type",
+    "op": "eq",
+    "value": "EngineBlock"
+}
+```
+
+### Name
+The name of the field you are using in this criteria.
+
+### Op
+The type of operation you are doing.
+
+### Value
+The value to compare with the operation.
+
+## db_mode
+This is optional.  It will defualt to 'sql' and usually should be but we have a 'monogo' mode as well.  You will likely ever need to tset this.
```

Comment:
Typos
`defualt` -> `default`
`monogo` -> `mongo`

and also

`You will likely ever need to tset this.`